src/types.ts

```diff
@@ -6,5 +6,10 @@ export type AResult = {
   robust: number;
   robustDeviation: number;
   highLimit: number;
-  lowLimit: number;  
-};
\ No newline at end of file
+  lowLimit: number;
+};
+export type ReferenceResult = {
+  value: number;
+  method: string;
+} | null;
+export type CochranResult = 'outlier' | 'straggler' | 'not-outlier' | null;
```

Comment:
Let's not build null into the type itself. Null is a type of its own. Whenever a function has the potential to return null, you can indicate that after the function's signature like so:

```ts
type Foo = {bar: number};

function Baz(): Foo | null {}
```